webapp/lib/app/nook/view.dart

```diff
@@ -269,6 +272,14 @@ class ConversationPanelView with AutomaticSuggestionIndicator {
 
     _messages = new DivElement()
       ..classes.add('messages');
+    _messages.onScroll.listen((e) {
+      if (_messages.scrollTop == _messages.scrollHeight - _messages.offsetHeight) {
+        _newMessageIndicator.classes.toggle("hidden", true);
+        _scrolledToBottom = true;
+      } else {
+        _scrolledToBottom = false;
+      }
```

Comment:
I think this could be simplified a bit:
```suggestion
      _scrolledToBottom = _messages.scrollTop == _messages.scrollHeight - _messages.offsetHeight;
      _newMessageIndicator.classes.toggle("hidden", _scrolledToBottom);
```

Reply:
🤯